src/main/java/org/fundacionjala/sfdc/pageobjects/SFDetails.java

```diff
@@ -11,7 +11,7 @@
 public abstract class SFDetails extends Base {
 
     @FindAll({
-            @FindBy(how = How.CSS, using = "topButtonRow.pbButton > input[name='edit']"),
+            @FindBy(how = How.CSS, using = "#topButtonRow input[name='edit']"),
```

Comment:
You need use @FindBy(css = "#topButtonRow input[name='edit']"), and without "#", please apply this on all FindBy's.